www_src/pages/discover/discover.jsx

```diff
@@ -1,42 +1,50 @@
-var React = require('react');
+var React = require('react/addons');
+var api = require('../../lib/api.js');
 var render = require('../../lib/render.jsx');
-var Binding = require('../../lib/binding.jsx');
 var Link = require('../../components/link/link.jsx');
-var api = require('../../lib/api.js');
 
 var Discover = React.createClass({
-  mixins: [Binding],
+  mixins: [],
   getInitialState: function () {
+    this.list = require('./mock.json');
+
     return {
-      test: false
+      list: this.list,
+      offset: 0,
+      limit: 10,
+      locale: {
+        language: 'en',
+        country: 'us'
+      }
     };
   },
-  componentWillMount: function () {
-    api({
-      uri: '/c0645e6953e9949f8e5c/raw/'
-    }, function (err, body) {
-      console.dir(err);
-      console.dir(body);
-    });
-  },
-  onClick: function () {
-    this.setState({test: !this.state.test});
-  },
   render: function () {
-    return (
-      <div id="discover" className="demo">
-        <Link url="/map/123" href="/pages/map" className="tile tile-demo">
-          <div className="inner">
-            <div className="img">
-              <img src="../../img/toucan.svg" />
-            </div>
+    var cards = [];
+    var project = null;
+    
+    for (var i = 0; i < this.state.list.length; i++) {
+      project = this.state.list[i];
+      console.dir(project.thumbnail[480]);
+      cards.push(
```

Comment:
this is a "bad plan" in terms of React. For React, you want to use `map` with key attributes instead:
```
cards = this.state.list.map( project => {
  var key = "something that uniquely identifies this project";
  return <Link ... key={key} ...> ... </Link>;
});
```